runhouse/main.py

```diff
@@ -112,46 +118,37 @@ def logout():
     raise typer.Exit()
 
 
-@app.command()
-def notebook(
-    cluster_name: str, up: bool = typer.Option(False, help="Start the cluster")
-):
+###############################
+# Cluster CLI commands
+###############################
+
+
+@cluster_app.command()
+def notebook(cluster_name: str):
     """Open a Jupyter notebook on a cluster."""
-    c = cluster(name=cluster_name)
-    if up:
-        c.up_if_not()
-    if not c.is_up():
-        console.print(
-            f"Cluster {cluster_name} is not up. Please run `runhouse notebook {cluster_name} --up` to bring "
-            f"it up if it is an on-demand cluster."
-        )
-        raise typer.Exit(1)
-    c.notebook()
+    current_cluster = get_cluster_or_local(cluster_name=cluster_name)
+    if not current_cluster.is_up():
+        print_bring_cluster_up_msg(cluster_name=cluster_name)
+        return
+    current_cluster.notebook()
 
 
-@app.command()
-def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluster")):
-    """SSH into a cluster created elsewhere (so `ssh cluster` doesn't work out of the box) or not yet up."""
+@cluster_app.command("ssh")
+def cluster_ssh(cluster_name: str):
+    """SSH into a cluster created elsewhere or not yet up.
```

Comment:
"or not yet up" isn't relevant anymore here right? cluster needs to be up for this to work